src/realty/domain/realty.spec.ts

```diff
@@ -1,104 +1,206 @@
 import { Realty } from './realty';
+import { RealtyEntityErrorCodes } from './realty-error-codes';
+import { RealtyType } from './realty-type';
 
 describe('Realty', () => {
-  it('should throw an error when title length is less than 3 characters', () => {
-    const expected_error = 'title must contain between 3 and 300 characters';
-    const shortTitle = 'ab';
-    expect(() =>
-      Realty.create({
-        title: shortTitle,
-      }),
-    ).toThrowError(expected_error);
-  });
+  describe('setTitle', () => {
+    describe('when valid title', () => {
+      it('should set title when title is valid', () => {
```

Comment:
```suggestion
    describe('when title is valid', () => {
      it('should set title', () => {
```
A condição já está no describe de cima